### PR TITLE
feat(desktop): update appimage installer script

### PR DESCRIPTION
**Pull request by doums**

### CODE REVIEW

File: nym-vpn-desktop/scripts/appimage.sh [hunk 1]
```diff
@@ -71,23 +71,59 @@ tilded() {
...
+<!DOCTYPE policyconfig PUBLIC
+ \"-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN\"
+ \"http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd\">
+<policyconfig>
+ <vendor>Nym</vendor>
+ <vendor_url>https://nymvpn.com</vendor_url>
+ <icon_name>nym-vpn</icon_name>
+ <action id=\"net.nymtech.nymvpn\">
+ <description>NymVPN</description>
+ <message>NymVPN requires root privileges for tunnel purposes</message>
```
Comment (octol) [inline nym-vpn-desktop/scripts/appimage.sh:116]: "NymVPN requires root privileges to create a local virtual network device and set system routing rules"

**Review by octol — APPROVED** [review]
I'm curious did you run this to something like shellcheck?

**Comment by doums** [comment]
@octol yes shellcheck & shfmt combo is the light in the darkness of bash xD

**Comment by tommyv1987** [comment]
resolved comments on the message - approving

**Pull request merged**